python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -177,20 +224,44 @@ def get_state(self):
             for k, v in state_dict.items():
                 state_dict[k] = v.cpu()
             cpu_state_dicts += [state_dict]
-        return {
+        return cpu_state_dicts
+
+    def _set_model_state_dicts(self, models_state_dicts):
```

Comment:
Should this be `_load_model_state_dicts` to match the call below?

Reply:
I think not - the trainer naming symmetry is "get_model_dict" and "set_model_dict". The underlying framework call is  `state_dict` -> `load_state_dict`.